ipmininet/router/config/zebra.py

```diff
@@ -121,19 +142,20 @@ def __init__(self, name=None, entries=()):
                         are composing the ACL"""
         AccessList.count += 1
         self.name = name if name else 'acl%d' % AccessList.count
-        self._entries = [e if isinstance(e, AccessListEntry)
-                         else AccessListEntry(prefix=e)
-                         for e in entries]
+        self.entries = [e if isinstance(e, AccessListEntry)
+                        else AccessListEntry(prefix=e)
+                        for e in entries]
 
-    def __iter__(self):
-        """Iterating over this ACL is basically iterating over all entries"""
-        return iter(self._entries)
+    # def __iter__(self):
+    #     """Iterating over this ACL is basically iterating over all entries"""
+    #     return iter(self.entries)
 
-    @property
-    def acl_type():
-        """Return the zebra string describing this ACL
-        (access-list, prefix-list, ...)"""
-        return 'access-list'
+
+#  @property
+#  def acl_type():
+#      """Return the zebra string describing this ACL
+#      (access-list, prefix-list, ...)"""
+#      return 'access-list'
 
 
 class RouteMapEntry(object):
```

Comment:
You seem to have removed RouteMapEntry to the RouteMap code, then you should remove dead code and check that Zebra template is still correct